docs/install/install-script.md

```diff
@@ -103,7 +107,9 @@ The `--namespace-scope-mode` will deploy `ServingRuntime`s confined to the same
 
 You can optionally provide a custom ModelMesh Serving image with `--modelmesh-serving-image`. If not specified, it will pull the latest image.
 
-The ModelMesh controller uses a webhook that requires a certificate. We suggest using [cert-manager](https://github.com/cert-manager/cert-manager) to provision the certificates for the webhook server. Other solutions should also work as long as they put the certificates in the desired location. You can follow [the cert-manager documentation](https://cert-manager.io/docs/installation/) to install it. If you don't want to install `cert-manager`, use the `--enable-self-signed-ca` flag. It will execute a script to create a self-signed CA and patch it to the webhook config.
+The ModelMesh controller uses a webhook that requires a certificate. We suggest using [cert-manager](https://github.com/cert-manager/cert-manager) to provision the certificates for the webhook server. Other solutions should also work as long as they put the certificates in the desired location. You can follow [the cert-manager documentation](https://cert-manager.io/docs/installation/) to install it. 
+
+If you don't want to install `cert-manager`, use the `--enable-self-signed-ca` flag. It will execute a script to generate the self-signed CA certificate and then restart the webhook pod to include the certificate in the config. Please install the [jq](https://jqlang.github.io/jq/download/) dependency before using the flag.
```

Comment:
```suggestion
If you don't want to install `cert-manager`, use the `--enable-self-signed-ca` flag. It will execute a script to generate the self-signed CA certificate and then restart the webhook pod to include the certificate in the config. This flag requires [jq](https://jqlang.github.io/jq/download/) to be installed.
```